pkg/admission/webhook/generic_webhook_test.go

```diff
@@ -303,16 +296,24 @@ func TestDispatch(t *testing.T) {
 	}
 }
 
-func toObjects(bindings []*apisv1alpha1.APIBinding) []runtime.Object {
```

Comment:
You could use generics to have a single implementation of this vs. creating a separate func

Reply:
Yeah; do we have an example for this in the codebase? I admit, I find Go generics a little confusing.

Reply:
Look at indexers.ByIndex